README.md

```diff
@@ -1,62 +1,86 @@
 # cookiecutter-awesome
-Cookiecutter to create an [awesome][awesome] list.
+Cookiecutter to create an [awesome][awesome-list] list.
 
+## Table of Contents
+
+- [Table of Contents](#table-of-contents)
 - [Features](#features)
 - [Requirements](#requirements)
 - [Usage](#usage)
-- [Documentation](#documentation)
+- [Example](#example)
 - [Development](#development)
-- [History](#history)
+  - [History](#history)
+  - [Community](#community)
+- [Other Awesome Lists](#other-awesome-lists)
 - [Credits](#credits)
 - [License](#license)
 
 ## Features
+
 - Creative Commons CC0 1.0 Universal License (as recommended in awesome's
-  [manifesto](https://github.com/sindresorhus/awesome/blob/master/awesome.md)).
+  [manifesto][manifesto]).
 - Travis configuration with [awesome_bot][awesome_bot] check.
 - Changelog, code of conduct, contributing, and pull request template files.
 - Compliant with awesome's [PR requirements][pr-requirements].
 
 ## Requirements
+
 - [git](https://git-scm.com/downloads)
 - [python](https://www.python.org/downloads/)
 - [cookiecutter](https://github.com/audreyr/cookiecutter)
 - ([awesome_bot][awesome_bot])
 
 ## Usage
+
 ```bash
-cookiecutter gh:Pawamoy/cookiecutter-awesome
+cookiecutter gh:moodule/cookiecutter-awesome
 # OR
-cookiecutter https://github.com/Pawamoy/cookiecutter-awesome
+cookiecutter https://github.com/moodule/cookiecutter-awesome
 ```
 
 Once your project has been generated:
 - Run `awesome_bot README.md` to verify the links.
 - Activate your repository in your Travis account.
 - **Please read [the PR requirements][pr-requirements]**
-  before submitting a Pull Request to [awesome][awesome].
+  before submitting a Pull Request to [awesome][awesome-list].
 
 ## Example
-See the [awesome-repository](https://github.com/Pawamoy/awesome-repository) list.
+
+See the [awesome-repository][awesome-list] list.
 
 ## Development
-See [CONTRIBUTING](CONTRIBUTING.md).
 
-## History
+Contributions welcome! Read the [contribution guidelines](CONTRIBUTING.md) first.
+
+### History
+
 See [CHANGELOG](CHANGELOG.md).
 
+### Community
+
+See [CODE OF CONDUCT](CODE_OF_CONDUCT.md)
+
+## Other Awesome Lists
+
+* [awesome-awesome](https://github.com/emijrp/awesome-awesome)
+* [awesome-awesomeness](https://github.com/bayandin/awesome-awesomeness)
+* [sindresorhus/awesome][awesome-list]
+* [The Warren](https://github.com/torchhound/warren)
+
 ## Credits
-This cookiecutter was created using contents from
-[cookie-cookie](https://github.com/tuxredux/cookie-cookie) and the
-[awesome-repository](https://github.com/Pawamoy/awesome-repository) list,
+
+This cookiecutter was created using contents from the
+[cookiecutter-awesome](https://github.com/Pawamoy/cookiecutter-awesome) template,
```

Comment:
I'd like to keep the credits for `cookie-cookie`. Also, this cookicutter was *not* generated with the yeoman generator. We can drop the "which itself was generated".

In the end, something like

```markdown
This cookiecutter was created using contents from the
[cookie-cookie template](https://github.com/tuxredux/cookie-cookie) and the
[Yeoman generator](https://github.com/dar5hak/generator-awesome-list).
```

would be better.

If you think it's worth it to keep credits for this very cookiecutter, I don't mind (but I find it a bit redundant haha).

Reply:
Sounds right!

This was my fork giving credit to you, but now that it's being merged we should keep it to the credits you gave :)

I lose myself a bit between the host repo, the generated repo, your version of the repo and my fork lol

Reply:
I lose myself too haha! How many times I've updated the templated contents / root contents instead of the other, I stopped counting :rofl: 

So, since it will be merged, and not a fork, no need to add credits for `pawamoy/cookiecutter-awesome`